<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(187, 253, 151);
font: 75 16pt &quot;FreeSerif&quot;;</string>
  </property>
  <widget class="QLabel" name="design_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;FreeSerif&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Игра окончена&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_done">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: rgb(131, 247, 125);
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
   </property>
   <property name="text">
    <string>Игра снова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_done_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: rgb(131, 247, 125);
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="win_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;FreeSerif&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Победил &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
